motor_control/throttle/base_throttle.h

```diff
@@ -1,21 +1,36 @@
 #ifndef BASE_THR_H
 #define BASE_THR_H
 
+//#include <Servo.h>
+#define throttleIn 5   // Placeholders
+#define ESCOutput 3
+#define button 7
+
 class Throttle
 {
 public:
-    Throttle(const int &pin);
```

Comment:
This is common behavior shared between all throttle implementations, so we absorb the responsibility of taking that pin and setting it as an input here. See the comment above on how to pass this value in through your inherited class's constructor.